certbot/display/util.py

```diff
@@ -126,9 +140,9 @@ def menu(self, message, choices, ok_label="OK", cancel_label="Cancel",
                 (str(i), choice) for i, choice in enumerate(choices, 1)
             ]
             # pylint: disable=star-args
-            code, index = self.dialog.menu(message, **menu_options)
+            code, index = _clean(self.dialog.menu(message, **menu_options))
 
-            if code == CANCEL:
+            if code == CANCEL or index == "":
```

Comment:
I can't remember why I added this now.  Perhaps I saw this for real, or perhaps I was just being defensive?